### PR TITLE
Doc - ssh hosts/nodes

### LINKED ISSUE #2711: Ability to SSH into hosts/nodes
## Description

Tanzu Community Edition is missing documentation on how to ssh into hosts/nodes

## Location

Administration

### PR DESCRIPTION
**Pull request by kcoriordan**
## What this PR does / why we need it
add doc for how to ssh into hosts/nodes

## Details for the Release Notes (PLEASE PROVIDE)

```none

```

## Which issue(s) this PR fixes

Fixes: #2711 

## Special notes for your reviewer
Run `hugo server `as per instructions in [readme](https://github.com/vmware-tanzu/community-edition/blob/main/docs/README.md)

### CODE REVIEW

**Comment by kcoriordan** [comment]
@alexnaris

File: docs/site/content/docs/latest/tips.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- vSphere management cluster and workload cluster nodes running on both Photon OS and Ubuntu: `capv`
+- Amazon EC2 bastion nodes: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Ubuntu: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Amazon Linux: `ec2-user`
+- Azure management cluster and workload cluster nodes (always Ubuntu): `capi`
+
+To connect to a node by using SSH, run one of the following commands from the machine that you use as the bootstrap machine:
+
+- vSphere nodes: `ssh capv@<em>node_address`
```
Comment (stmcginnis) [inline docs/site/content/docs/latest/tips.md:15]: What does the `<em>` mean here? If this is an HTML emphasis tag, that will not be formatted correctly in inside the code formatting markdown.

Should these also includer `-i [path_to_key_file]`?

File: docs/site/content/docs/latest/tips.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Connect to Cluster Nodes with SSH
+
+You can use SSH to connect to individual nodes of management clusters or workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command. Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.
```
Comment (joshrosso) [inline docs/site/content/docs/latest/tips.md:3]:
```suggestion
You can use SSH to connect to nodes in management clusters and workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command.
```

Comment (joshrosso) [reply docs/site/content/docs/latest/tips.md:3]: some word smithing and recommending the removal of:

>  Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.

This is not required, only the ssh key being on the client (host) you're connecting **from**.

File: docs/site/content/docs/latest/tips.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+The SSH keys that you register with the management cluster, and consequently that are used by any workload clusters that you deploy from the management cluster, are associated with the following user accounts:
```
Comment (joshrosso) [inline docs/site/content/docs/latest/tips.md:5]:
```suggestion
The SSH key used to access clusters are associated with the following Linux users:
```

File: docs/site/content/docs/latest/tips.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- vSphere management cluster and workload cluster nodes running on both Photon OS and Ubuntu: `capv`
+- Amazon EC2 bastion nodes: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Ubuntu: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Amazon Linux: `ec2-user`
+- Azure management cluster and workload cluster nodes (always Ubuntu): `capi`
+
+To connect to a node by using SSH, run one of the following commands from the machine that you use as the bootstrap machine:
```
Comment (joshrosso) [inline docs/site/content/docs/latest/tips.md:13]:
```suggestion
To connect to a node by using SSH, run one of the following commands from a machine containing the SSH key:
```

File: docs/site/content/docs/latest/tips.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- vSphere nodes: `ssh capv@<em>node_address`
+- Amazon EC2 bastion nodes, management cluster, and workload nodes on Ubuntu: `ssh ubuntu@<em>node_address`
+- Amazon EC2 management cluster and workload nodes running on Amazon Linux: `ssh ec2-user@<em>node_address`
+- Azure nodes: `ssh capi@<em>node_address`
+
+Because the SSH key is present on the system on which you are running the `ssh` command, no password is required.
```
Comment (joshrosso) [inline docs/site/content/docs/latest/tips.md:20]:
```suggestion
> Each cluster host contains the public key. The hosts will not accept an SSH password (`PasswordAuthentication` is set to `no`). 
```

**Review by joshrosso — CHANGES_REQUESTED** [review]
looking forward to getting this in, requested changes.

File: docs/site/content/docs/latest/tips.md (hunk 1 above)
Comment (dvonthenen) [reply docs/site/content/docs/latest/tips.md:15]: I think we need an update here... based on feedback

Comment (kcoriordan) [reply docs/site/content/docs/latest/tips.md:15]: removed the html
I'm not sure about adding -i [path_to_key_file], I think the key just needs to be on the machine

**Comment by kcoriordan** [comment]
I think this is ready to merge.

**Pull request merged**